charts/mattermost-chaos-engine/values.yaml

```diff
@@ -0,0 +1,90 @@
+# Default values for mattermost-chaos-engine.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: mattermost/mattermost-app-chaosengine
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```

Comment:
@spirosoik Ideally we don't want latest tag.

Reply:
as default value it's fine right? typically it shouls respect chart app version if you don't pass the tag at all